src/mobile-app/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Mobile App</title>
+    <title>Monitor Your World</title>
     <meta name="description" content="Mobile App">
```

Comment:
Is it ok to leave the description generic too?

Reply:
I guess it is - the app passed the release checks with this unchanged, though future work may want to change it.